webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTable.js

```diff
@@ -18,7 +18,7 @@ const ModelsTable = ({
 }) => {
   useEffect(() => {
     getTableItems(getURIQuery(window.location.href));
-  }, []);
+  }, [getTableItems]);
```

Comment:
Glad to see the Hooks eslint rules finally made it in :)

Reply:
Looks like this same `getTableItems` dependency will also require a PR in `foreman-tasks`; not sure if that exists already
```
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/foreman-tasks-0.17.4/webpack/ForemanTasks/Components/TasksTable/TasksTable.js
  34:6  warning  React Hook useEffect has missing dependencies: 'getTableItems' and 'url'. Either include them or remove the dependency array. If 'getTableItems' changes too often, find the parent component that defines it and wrap that definition in useCallback  react-hooks/exhaustive-deps

✖ 1 problem (0 errors, 1 warning)
```

Reply:
Should we add `getURIQuery(window.location.href)` to the array as well?
So it will fetch again when the URL changes.

Reply:
@sharvit yeah probably